go.mod

```diff
@@ -1,6 +1,8 @@
 module sigs.k8s.io/ibm-powervs-block-csi-driver
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
Can you please delete this line? This creates an issue for sec scans in one of the downstream projects.

Reply:
Done, thanks!